qml/StatisticsChartView.qml

```diff
@@ -251,8 +257,33 @@ ChartView {
     function setYAxis(min, max, niceNumbers = true, force = false) {
         // If axes has been set manually and forced is not set, do not change
         if (force || !manuallySetAxes) {
-            axisY.min = min
-            axisY.max = max
+            if (min == y_MIN_DEFAULT || min == y_MAX_DEFAULT) 
+            {
+                axisY.min = 0
+            }
+            else
+            {
+
+                axisY.min = min
+            }
+            if (max == y_MIN_DEFAULT || max == y_MAX_DEFAULT) 
+            {
+                axisY.max = 1
+            }
+            else
+            {
+                axisY.max = max
+            }
+            if (axisY.min == axisY.max) 
+            {
+                axisY.min = min - 1
+                axisY.max = max + 1
+            }
+            if (axisY.min > axisY.max) 
```

Comment:
```suggestion
            else if (axisY.min > axisY.max) 
```
this could never happen at the same time of above call, and this would (slightly) speed up